wagtailio/static/sass/components/_footer-menu.scss

```
@@ -120,6 +120,21 @@
         }
     }
 
+    &__logo {
+        display: none;
+
+        @include media-query(large) {
+            margin-top: auto;
+            width: 155px;
+            height: 65px;
+            display: block;
+        }
+
+        @media (forced-colors: active) {
+            fill: $color--white; // force white if in high contrast mode
```

```suggestion
            fill: CanvasText;
```

Just using white will work with a dark theme but not with a light theme. We need to use a system color instead.